custom-metrics-stackdriver-adapter/pkg/adapter/translator/query_builder_test.go

```diff
@@ -779,7 +788,47 @@ func TestTranslator_GetSDReqForNodes(t *testing.T) {
 	}
 }
 
-func TestTranslator_GetSDReqForNodes_withMetricSelector(t *testing.T) {
+func TestQueryBuilder_Multiple_Node(t *testing.T) {
```

Comment:
Nit: multiple nodes

Reply:
yea, I double considered the naming as well, `Node` here I used singular refers to the type node, but definitely I'm good to change it to `nodes`

Reply:
What is Multiple for here? Multiple filters or multiple nodes?